<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemPedido</class>
 <widget class="QDialog" name="ItemPedido">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_11">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout"/>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QStackedWidget" name="stItemPedido">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="gridLayout_2">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Id Produto</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>Código de barras</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>Descrição</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="0" column="1">
                   <layout class="QGridLayout" name="gridLayout_3">
                    <item row="0" column="0" rowspan="3">
                     <layout class="QFormLayout" name="formLayout">
                      <item row="2" column="0" colspan="2">
                       <widget class="QLineEdit" name="lE_descricao"/>
                      </item>
                      <item row="0" column="0" colspan="2">
                       <widget class="QLineEdit" name="lE_IdPRoduto"/>
                      </item>
                      <item row="1" column="0" colspan="2">
                       <widget class="QLineEdit" name="lE_codBarras"/>
                      </item>
                     </layout>
                    </item>
                    <item row="0" column="1">
                     <widget class="QPushButton" name="pB_PesqIdProdt">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</normaloff>
                        <disabledoff>:/rec/icons/pesquisar.png</disabledoff>
                        <selectedoff>:/rec/icons/pesquisar.png</selectedoff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</iconset>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QPushButton" name="pB_PesqCodigo">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</normaloff>
                        <disabledoff>:/rec/icons/pesquisar.png</disabledoff>
                        <selectedoff>:/rec/icons/pesquisar.png</selectedoff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</iconset>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QPushButton" name="pB_PesqDescr">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</normaloff>
                        <disabledoff>:/rec/icons/pesquisar.png</disabledoff>
                        <selectedoff>:/rec/icons/pesquisar.png</selectedoff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</iconset>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <widget class="QTreeView" name="treeView"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QTableView" name="tv_itemPedido"/>
                  </item>
                  <item row="2" column="0">
                   <layout class="QGridLayout" name="gridLayout_7">
                    <item row="0" column="0">
                     <layout class="QGridLayout" name="gridLayout_6">
                      <item row="0" column="0">
                       <widget class="QLabel" name="label_4">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="text">
                         <string>Qtd</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1">
                       <widget class="QSpinBox" name="spinBox">
                        <property name="maximum">
                         <number>9999</number>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="2">
                       <widget class="QPushButton" name="PbRemover">
                        <property name="text">
                         <string>Remover</string>
                        </property>
                        <property name="autoDefault">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="3">
                       <spacer name="horizontalSpacer_2">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </item>
                    <item row="0" column="1">
                     <widget class="QPushButton" name="pBFormadPagamento">
                      <property name="text">
                       <string>Forma de pagamento</string>
                      </property>
                      <property name="autoDefault">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
